libs/diagnostics/performance_handler.hpp

```diff
@@ -42,30 +42,23 @@ enum class Unit {
   COUNT
 };
 
+using namespace boost::histogram;
```

Comment:
Can you please delete this line and explicitly use `boost::histogram` ?

Reply:
yes made the changes.